src/NServiceBus.Transport.RabbitMQ/Administration/ManagementClient/Models/QueueArguments.cs

```diff
@@ -12,11 +12,11 @@ class QueueArguments
     [JsonConverter(typeof(QueueTypeConverter))]
     public QueueType? QueueType { get; set; }
 
-    [JsonPropertyName("x-delivery-limit")]
+    [JsonPropertyName("delivery_limit")]
```

Comment:
I'm not sure this change is correct.  The best doc I can find is the last paragraph in the [Quorum Queues in the Management](https://www.rabbitmq.com/blog/2020/04/20/rabbitmq-gets-an-ha-upgrade#quorum-queues-in-the-management-ui) UI section

Reply:
This has been reverted